packages/storybook-aem/app/cmds/package/install.js

```diff
@@ -1,39 +1,53 @@
 const fs = require('fs');
 const path = require('path');
 const FormData = require('form-data');
-const { exec } = require('child_process');
-
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const chalk = require('chalk');
+const zip = require('zip-promise');
 const cwd = process.cwd();
 const log = require('../../utils/logger');
 const fetchFromAEM = require('../../utils/fetchFromAEM');
 
 module.exports = async (args,config) => {
-    log(`Checking for Storybook AEM Content Package configuration...`);
     const packageManagerURL = `/crx/packmgr/service.jsp`;
-    const storybookPackagePath = path.resolve(cwd, config.projectRoot, config.relativeProjectRoot, config.localPackagePath, config.packageName);
-    if (!fs.existsSync(storybookPackagePath)) {
-        throw log(chalk.red(`error, could not find Storybook AEM Content Package`));
-    }
+    const storybookPackageDirectory = path.resolve(cwd, config.projectRoot, config.relativeProjectRoot, config.localPackagePath);
+    const storybookPackagePath = path.resolve(storybookPackageDirectory, config.packageName);
+
+    try {
+      log(`Zipping Storybook library ...`);
+      await zip.folder(storybookPackageDirectory, storybookPackagePath);
+
+      log(`Checking for Storybook AEM Content Package ...`);
+      if (!fs.existsSync(storybookPackagePath)) {
+          throw log(chalk.red(`error, could not find Storybook AEM Content Package`));
+      }
```

Comment:
Should this fs.existsSync be ran before the zipping?

Reply:
Just checked on this. The zipping is what creates it, so the `existsSync` is really just validating that the zip worked.